src/pages/SetupDatabase.vue

```diff
@@ -151,7 +151,7 @@ export default {
                 await sleep(2000);
                 await this.goToMainServerWhenReady();
             } catch (e) {
-                toast.error(e.response.data);
+                this.$root.toastError(e.response.data);
```

Comment:
One specific return string would require interpolation: 

```js
"Cannot connect to the database: " + e.message
```

I think using something like `toastRes` would be better here.

Reply:
May need some work, since error messages in this page cannot fit into `toastRes`. Switching back to `toast.error()` first